docs/contributing/contributing.md

```diff
@@ -0,0 +1,83 @@
+# Contributing to Labelmerge
+
+Labelmerge python package dependencies are managed with Poetry (`v1.2.0+`), which
+you will need installed on your machine. You can find installation instructions 
+on the [Poetry website](https://python-poetry.org/docs/master/#installation).
+
+We **strongly** recommend using Labelmerge with the `--use-singularity` flag, which 
```

Comment:
I would soften this a fair bit because if we deploy to PyPI people can just `pip install` this.